macro/vignettes/simpletrip.Rmd

```diff
@@ -33,7 +33,7 @@ $$
 
 In this set up, $S_{1}$ is the state where the resident of patch 1 is at home but resident of patch 2 is away. $S_{2}$ is where neither is home. $S_{3}$ is where both are home. $S_{4}$ is where the resident of patch 1 is away but resident of patch 2 is home.
 
-We can heuristically derive the master equations to describe this stochastic model by remembering that as $dt\rightarrow 0$ only single particle jumps are allowed.
+We can heuristically derive the master equations to describe this stochastic model by remembering that as $dt\rightarrow 0$ only no more than one event is allowed to happen at a time. This means that the master equations can't allow, for example, for resident 1 to take a trip and resident 2 to go home at the exact same point in time.
```

Comment:
This vignette is grand. Why am I seeing only part of it in the pull request? Did we commit the rest earlier? Odd. Anyhow, nice to see the model explained.

Reply:
yep! we committed the rest earlier. I was just clarifying this specific sentence based on feedback from Falko, after he read the version I posted in code-dev channel.